sql/hive/src/main/scala/org/apache/spark/sql/hive/TableReader.scala

```diff
@@ -79,8 +79,9 @@ class HadoopTableReader(
   private val _minSplitsPerRDD = if (sparkSession.sparkContext.isLocal) {
     0 // will splitted based on block by default.
   } else {
-    math.max(hadoopConf.getInt("mapreduce.job.maps", 1),
-      sparkSession.sparkContext.defaultMinPartitions)
+    val value = sparkSession.sessionState.conf.hiveMinPartitionNum
+      .getOrElse(sparkSession.sparkContext.defaultMinPartitions)
+    math.max(hadoopConf.getInt("mapreduce.job.maps", 1), value)
```

Comment:
Could we get the configuration value from `sparkSession.sessionState.conf` instead of `hadoopconf`?

Reply:
This is also okay, simply discard the configuration obtained from hadoopconf, but for compatibility, this user has already been configured in hadoopconf. I think it can also be added here

Reply:
@wangyum Hi, sir, can you initiate another test? These errors seem unrelated to my current modifications

Reply:
@wangyum  Hi, sir, are you still following